src/components/PathwaysList/PathwayModal.tsx

```diff
@@ -16,18 +16,21 @@ import shortid from 'shortid';
 import { usePathwayContext } from 'components/PathwayProvider';
 import useStyles from './styles';
 import { createNewPathway } from 'utils/builder';
+import { Pathway } from 'pathways-model';
 
-interface NewPathwayModalProps {
+interface PathwayModalProps {
   open: boolean;
   onClose: () => void;
+  editPath?: Pathway;
 }
 
-const NewPathwayModal: FC<NewPathwayModalProps> = ({ open, onClose }) => {
+const PathwayModal: FC<PathwayModalProps> = ({ open, onClose, editPath }) => {
+  const createNewModal = !editPath;
```

Comment:
Suggest this be renamed to `createNewPathway` since the boolean is whether or not we are creating a new pathway not a new modal

Reply:
Used createNewPathwayMeta instead because createNewPathway is a function call